### DISCUSSION TITLE
Multiple Topology Views

### DISCUSSION THREAD

**Discussion by jdtinsley88** [mattieserver/netbox-topology-views#204] [Q&A]
Maybe I am over looking something, but how do you create multiple topology views easily?

**Comment by mattieserver** [comment]
Hi,

You cannot, well you can but you have to change the filters every time.

There is no way to save the topology in netbox itself(or the plugin).
You could however copy the url and add that link to your documentation since all the filters are in the URL.

**Comment by ekrichbaum** [comment]
What about a panel in the region/site/location page that displayed the topology with the region/site/location/ as the filter?  I saw some old plugins that did that or a button to open a popup with the view (with the appropriate filter).  There isn't a filter by rack but that could be a potential view point as well.

**Comment by joaolucasmacedo** [comment]
> Maybe I am over looking something, but how do you create multiple topology views easily?

I just use TAGs to do so. I just create tags for each topology, and than it becomes very simple to select any topology with desired devices.
